engine/src/main/java/org/terasology/logic/players/PlayerSystem.java

```diff
@@ -70,13 +71,19 @@
 @RegisterSystem(RegisterMode.AUTHORITY)
 public class PlayerSystem extends BaseComponentSystem implements UpdateSubscriberSystem {
 
+    private static final Logger logger = LoggerFactory.getLogger(PlayerSystem.class);
+
+    private final float playerHeight = 1.5f;
```

Comment:
I see this got moved from spawnPlayer - any chance we can just load it from the player prefab?